script/utils/transferOwnership.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+
+# Default to dry run mode
+dry_run=true
+
+# Check if required environment variables are set
+if [ -z "$TARGET_RPC_URL" ] || [ -z "$OWNER_ADDRESS" ] || [ -z "$UPGRADER_ADDRESS" ] || [ -z "$EORACLE_CHAIN_ID" ] || [ -z "$DEPLOYER_PRIVATE_KEY" ]; then
+    echo "Error: Required environment variables not set"
+    echo "Please set TARGET_RPC_URL, OWNER_ADDRESS, UPGRADER_ADDRESS, EORACLE_CHAIN_ID, and DEPLOYER_PRIVATE_KEY"
+    exit 1
+fi
```

Comment:
use the same format as operator-management scripts in tools repo to pass parameters to the script